### PR TITLE
chore: update ce sdk version and codegen

### PR DESCRIPTION
**Pull request by Cali0707**
Updates the cloudevents sdk to support CESQL v1

## Proposed Changes

- Update the cloudevents sdk
- Update codegen (this led to an unrelated format change)

**Release Note**

```release-note
The CESQL filter dialect now supports the CESQL v1 specification (https://github.com/cloudevents/spec/releases/tag/cesql%2Fv1.0.0)
```

### CODE REVIEW

File: data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/filter/subscriptionsapi/CeSqlFilter.java [hunk 1]
```diff
@@ -17,11 +17,12 @@
 
 import dev.knative.eventing.kafka.broker.dispatcher.Filter;
 import io.cloudevents.CloudEvent;
-import io.cloudevents.sql.EvaluationException;
-import io.cloudevents.sql.EvaluationRuntime;
-import io.cloudevents.sql.Expression;
-import io.cloudevents.sql.Parser;
-import io.cloudevents.sql.Type;
+import io.cloudevents.sql.*;
```
Comment (matzew) [inline data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/filter/subscriptionsapi/CeSqlFilter.java:20]: no no. no wildcards :-)

File: data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/impl/filter/subscriptionsapi/CeSqlFilterTest.java [hunk 2]
```diff
@@ -51,7 +51,7 @@ static Stream<Arguments> testCases() {
                 Arguments.of(event, "'TRUE'", true),
                 Arguments.of(event, "'FALSE'", false),
                 Arguments.of(event, "0", false),
-                Arguments.of(event, "1", false),
+                Arguments.of(event, "1", true),
```
Comment (matzew) [inline data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/impl/filter/subscriptionsapi/CeSqlFilterTest.java:54]: wat?

Comment (matzew) [reply data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/impl/filter/subscriptionsapi/CeSqlFilterTest.java:54]: why was that false before? :sweat_smile:

Comment (Cali0707) [reply data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/impl/filter/subscriptionsapi/CeSqlFilterTest.java:54]: Oh before integer to boolean casting was a cast error, which turns in a filter result of false. In v1, we defined integer to boolean casting so now it works :)

**Review by matzew — APPROVED** [review]
/lgtm
/approve

File: data-plane/THIRD-PARTY.txt [hunk 3]
```diff
@@ -32,6 +32,7 @@ Lists of 245 third-party dependencies.
      (Apache License 2.0) Metrics Core Library (com.yammer.metrics:metrics-core:2.2.0 - http://metrics.codahale.com/metrics-core/)
      (Apache License, Version 2.0) Apache Commons BeanUtils (commons-beanutils:commons-beanutils:1.9.4 - https://commons.apache.org/proper/commons-beanutils/)
      (Apache License, Version 2.0) Apache Commons CLI (commons-cli:commons-cli:1.4 - http://commons.apache.org/proper/commons-cli/)
+     (Apache-2.0) Apache Commons Codec (commons-codec:commons-codec:1.16.1 - https://commons.apache.org/proper/commons-codec/)
      (Apache-2.0) Apache Commons Codec (commons-codec:commons-codec:1.17.0 - https://commons.apache.org/proper/commons-codec/)
```
Comment (pierDipi) [inline data-plane/THIRD-PARTY.txt:36]: same library 2 different versions :confused:

Comment (Cali0707) [reply data-plane/THIRD-PARTY.txt:36]: Oh not sure how that happened. Any ideas?

**Pull request merged**